openedx/core/djangoapps/discussions/views.py

```diff
@@ -42,6 +63,78 @@ class Meta:
         ]
 
 
+class LegacySettingsSerializer(serializers.BaseSerializer):
+    """
+    Serialize legacy discussions settings
+    """
+    class Meta:
+        fields = [
```

Comment:
Does this list line up with the fields expected in the new UI?  I feel like something is missing (am on my phone and on PTO or would look)

Reply:
@davidjoy Ah yeah, I overlooked the cohort-related settings; I was going off an older mock/iteration.
As it turns out, there's already an HTTP API for those settings (with custom serializers, etc.) that is used to drive the existing experience (via the instructor dashboard).

I took a quick look at the underlying code; it seems like some work to shoe-horn that API into ours.
Depending on the frontend arch, I'm wondering if it's easy enough of a lift to just hit this existing LMS endpoint directly from the MFE.

There's probably still a path forward handling it all on the backend, and I do like the idea of a single endpoint from "the beginning", but I'm hoping it's easy enough to patch in (literally, `HTTP PATCH` :P ) to the existing API :crossed_fingers: 

Happy to cover more when you're back "in office" :)